modules/repository-datastax/src/main/java/com/intuit/wasabi/repository/cassandra/impl/CassandraExperimentRepository.java

```diff
@@ -97,6 +100,8 @@
 
     private ExperimentTagAccessor experimentTagAccessor;
 
+    private PrioritiesAccessor prioritiesAccessor;
```

Comment:
@abhishekjain88 I would prefer to declare it as final rather than adding getter/setters as it just clouds code unnecessary... May be in some other card 😄